src/ButtonComponent.cpp

```diff
@@ -55,3 +57,19 @@ void ButtonComponent::paint(Graphics &g)
 void ButtonComponent::paintButton(Graphics &, bool, bool)
 {
 }
+
+void ButtonComponent::set_options(uint8_t value)
+{
+	// adjust the position of the childs to the button face area if the NoBorder attribute is changed
+	if ((value & static_cast<uint8_t>(Options::NoBorder)) != get_option(Options::NoBorder))
+	{
+		auto borderOffset = (value & static_cast<uint8_t>(Options::NoBorder)) ? -4 : 4;
```

Comment:
Nit, implicit conversion from u8 to bool

Maybe something like this would be better?
```suggestion
		auto borderOffset =(0 != (value & static_cast<uint8_t>(Options::NoBorder))) ? -4 : 4;
```

Reply:
> Nit, implicit conversion from u8 to bool
> 
> Maybe something like this would be better?

Hi @ad3154 

Do you want to do a next review round after the fix, or I can merge this one?